### DISCUSSION TITLE
Allow masking fields from appearing in user data

### DISCUSSION THREAD

**Discussion by fubuloubu** [fief-dev/fief#390] [Ideas]
In the examples there is an `agree_to_terms` field used to demonstrate how more mechanical fields can work. However, this isn't really a field that any user (including `current_user`) needs to see, more of an "internal" field (similar to an "allow marketing" boolean) for use with other products (aka clients).

TL;DR - It would be useful to be able to "hide" a user field from appearing in `/userinfo`

**Comment by frankie567** [comment]
Interesting! Added it to the backlog :)

**Reply by frankie567** [reply]
https://github.com/orgs/fief-dev/projects/1/views/4?pane=issue&itemId=70860532
